.github/workflows/mark-and-close-stale-issues.yml

```
@@ -0,0 +1,17 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          stale-issue-message: 'This issue was marked as stale and going to be closed soon'
+          stale-pr-message: 'This PR was marked as stale and going to be closed soon'
```

open to different wording, but I'd like to communicate how the user can continue.  For airbyte employees I don't really care too much.  They are adults who can ask for code reviews.  My target audience here is more an OSS contributor who is giving us their time and energy for free.  How do we clearly tell them why we are closing the issue and what they can do if they want to continue
```suggestion
          stale-pr-message: 'This PR was marked as stale due to inactivity and going to be closed soon.  You may re-open the PR if you still want to merge it.  Please consider tagging another reviewer if you are having trouble getting a review'
```